tests/remote_point_evaluation/mapping_04.cc

```diff
@@ -42,7 +42,12 @@ do_test(const bool                     enforce_unique_map,
 
   MappingQ1<dim> mapping;
 
-  Utilities::MPI::RemotePointEvaluation<dim> eval(1.e-6, enforce_unique_map);
+  typename Utilities::MPI::RemotePointEvaluation<dim>::AdditionalData
+    additional_data;
+  additional_data.enforce_unique_mapping = enforce_unique_map;
+  additional_data.tolerance              = 1e-6;
```

Comment:
Do you need to manually set the tolerance since this is the default value of the tolerance in the additional data struct? or are you doing this for just clarity's sake?

Reply:
I just tried to set the values accordingly in the test cases. However, I agree that in this case with the new additional data struct we could skip the explicit setter.